include/gz/rendering/Image.hh

```diff
@@ -48,8 +46,26 @@ namespace gz
       public: Image(unsigned int _width, unsigned int _height,
                   PixelFormat _format);
 
+      /// \brief Copy Constructor
+      /// \param[in] _other The other image
+      public: Image(const Image &_other);
```

Comment:
I think if you use `ImplPtr` rather than `UniqueImplPtr`, you will get all of these functions for "free" and can remove them here.  Looking at your copy constructor, nothing crazy is happening (just copying the pointer), so I think you would be fine with the shared approach.

Reply:
ah that's what I was looking for! I knew there's something that does this already. Updated to use `GZ_UTILS_IMPL_PTR`. [c891cd0](https://github.com/gazebosim/gz-rendering/pull/875/commits/c891cd0755d49431bf0870673a550108c21bfdcc)